docs/tutorials/visualization.rst

```diff
@@ -78,12 +78,6 @@ The plotter that generated the figure can be accessed through the analysis insta
 and customizing the figure can be done by setting the plotter's options. We now modify
 the color, symbols, and size of our plot, as well as change the axis labels for the amplitude units:
 
-.. jupyter-execute::
-    :hide-code:
-    :hide-output:
-
-    %matplotlib inline
-
```

Comment:
Not sure why, but removing this block got rid of the `RuntimeWarning: divide by zero encountered in det` numpy errors for me locally. The block also doesn't do anything to begin with since the figures already render fine.

Reply:
Strange.